src/status_im/ui2/screens/chat/components/contact_item/view.cljs

```diff
@@ -1,29 +1,32 @@
 (ns status-im.ui2.screens.chat.components.contact-item.view
-  (:require [status-im.utils.handlers :refer [<sub >evt]]
-            [quo2.foundations.typography :as typography]
+  (:require [quo2.foundations.typography :as typography]
             [quo2.components.icon :as icons]
             [quo2.foundations.colors :as colors]
             [quo2.components.avatars.user-avatar :as user-avatar]
             [quo.react-native :as rn]
-            [status-im.utils.utils :refer [get-shortened-address]]
+            [status-im.utils.utils :as utils]
             [quo.platform :as platform]
             [quo2.components.markdown.text :as text]
-            [status-im.ui2.screens.chat.components.message-home-item.style :as style]))
+            [status-im.ui2.screens.chat.components.message-home-item.style :as style]
+            [status-im.utils.re-frame :as rf]
+            [status-im.ui2.screens.chat.actions :as actions]))
 
 (defn open-chat [chat-id]
-  (>evt [:dismiss-keyboard])
+  (rf/dispatch [:dismiss-keyboard])
   (if platform/android?
-    (>evt [:chat.ui/navigate-to-chat-nav2 chat-id])
-    (>evt [:chat.ui/navigate-to-chat chat-id]))
-  (>evt [:search/home-filter-changed nil])
-  (>evt [:accept-all-activity-center-notifications-from-chat chat-id]))
+    (rf/dispatch [:chat.ui/navigate-to-chat-nav2 chat-id])
+    (rf/dispatch [:chat.ui/navigate-to-chat chat-id]))
+  (rf/dispatch [:search/home-filter-changed nil])
+  (rf/dispatch [:accept-all-activity-center-notifications-from-chat chat-id]))
 
 (defn contact-item [item]
   (let [{:keys [public-key ens-verified added? images]} item
-        display-name (first (<sub [:contacts/contact-two-names-by-identity public-key]))
-        photo-path   (if-not (empty? images) (<sub [:chats/photo-path public-key]) nil)]
-    [rn/touchable-opacity (merge {:style   (style/container)
-                                  :on-press #(open-chat public-key)})
+        display-name (first (rf/sub [:contacts/contact-two-names-by-identity public-key]))
+        photo-path   (if-not (empty? images) (rf/sub [:chats/photo-path public-key]) nil)]
```

Comment:
Since you touched this line, you can also replace `if-not` for `when-not`. `(if-not x something-true nil)` is never necessary in Clojure. Or, better yet, if `images` is a sequence:

```clojure
(when (seq images) (rf/sub [:chats/photo-path public-key]))
```